source/common/crypto/utility.cc

```diff
@@ -25,8 +25,7 @@ std::vector<uint8_t> Utility::getSha256Digest(const Buffer::Instance& buffer) {
     rc = EVP_DigestUpdate(ctx, slice.mem_, slice.len_);
     RELEASE_ASSERT(rc == 1, "Failed to update digest");
   }
-  unsigned int len;
-  rc = EVP_DigestFinal(ctx, digest.data(), &len);
+  rc = EVP_DigestFinal(ctx, digest.data(), NULL);
```

Comment:
s/`NULL`/`nullptr`

Reply:
this is calling C functions... that's why i sticked with `NULL`.